exercises/F-logical-operators/exercise.js

```diff
@@ -11,14 +11,14 @@ var cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-var htmlLevelAbove5;
-var cssLevelAbove5;
+var htmlLevelAbove5 = htmlLevel > 5;
+var cssLevelAbove5 = cssLevel > 5;
 
 // Finish the next two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
-var cssAndHtmlAbove5;
-var cssOrHtmlAbove5;
+var cssAndHtmlAbove5 = htmlLevel > 5 && cssLevel > 5;       /* htmlLevel && cssLevel > 5 */
+var cssOrHtmlAbove5 = htmlLevel > 5 || cssLevel > 5;        /* htmlLevel || cssLevel > 5 */
```

Comment:
For this exercise, we'll like you to use the variables on line 14 and 15 that you defined. How would you write your expressions using `htmlLevelAbout5` and `cssLevelAbove5`?

Reply:
@louisechow I wrote the expression as the following : 

```
var cssAndHtmlAbove5 = htmlLevelAbove5 && cssLevelAbove5;      
var cssOrHtmlAbove5 = htmlLevelAbove5 || cssLevelAbove5;  
```